Add nixpkgs bot (#180)
nix-community/awesome-nix: README.md

* [Nix Infra Status](https://status.nixos.org) - Get the age and current git commit of each Nix channel.
* [Nix Review Tools Reports](https://malob.github.io/nix-review-tools-reports/) - Reports showing problematic dependencies (dependencies causing the most failed builds) for major Hydra jobsets.
* [nixpkgs PR tracker](https://nixpk.gs/pr-tracker.html) - A tracker for whether a PR has made it into a channel yet.

> * [Nixpkgs Bot](https://git.maralorn.de/nixos-config/tree/packages/nixpkgs-bot) - A Matrix bot to track/subscribe to PR to know when they are merged and eaches relevant branches.

Reviewer: ```suggestion
* [Nixpkgs Bot](https://git.maralorn.de/nixos-config/tree/packages/nixpkgs-bot) - A Matrix bot to track Nixpkgs Pull Requests by target branch.
```

Language could use some cleanup, but the suggestion above I believe retains the meaning

Reviewer: Yeah, that's good. Will leave committing to @j4m3s-s though
